WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -102,6 +103,7 @@
     @Inject SiteStore mSiteStore;
     @Inject PostStore mPostStore;
     @Inject Dispatcher mDispatcher;
+    @Inject protected LoginAnalyticsListener mAnalyticsListener;
```

Comment:
Since this is not a login-related class it would probably be clearer to call this `mLoginAnalyticsListener` here. We might also want to change it in the other classes in the `WordPress` module (keeping it `mAnalyticsListener`  inside the login library though does make sense IMO since it's the only analytics pathway there).

Reply:
I update the variable name in 591c9c3.